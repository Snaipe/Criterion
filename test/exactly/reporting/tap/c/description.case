[act]

% description.c.bin --tap

[assert]

stdout
  -transformed-by replace_version
  equals -contents-of expected/description.txt
