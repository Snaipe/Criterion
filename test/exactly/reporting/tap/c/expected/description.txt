TAP version 13
1..2
# Criterion <VERSION>
#
# Running 2 tests from misc
ok - misc::skipped This one is skipped # SKIP test was skipped
not ok - misc::failing Just a failing test
  ---
  assertions: 1
  failures:
  - description.c:4: |+
      Assertion failed: The expression 0 is false.
  ...
